partner_second_lastname/static/description/icon.svg

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Using default OCA icon you don't need to add the svg we already have it in another repo